src/main/java/com/snack/news/util/SlackAuthHttpClient.java

```diff
@@ -0,0 +1,59 @@
+package com.snack.news.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.snack.news.domain.slack.SlackChannel;
+import org.springframework.stereotype.Component;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Map;
+
+@Component
+public class SlackAuthHttpClient {
+
+	private static RestTemplate slackHttpClient;
+
+	public SlackAuthHttpClient(RestTemplate slackHttpClient) {
+		SlackAuthHttpClient.slackHttpClient = slackHttpClient;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=5471424)